modules/nf-core/agat/spaddintrons/main.nf

```diff
@@ -0,0 +1,48 @@
+process AGAT_SPADDINTRONS {
+    tag '$gff'
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/agat:1.2.0--pl5321hdfd78af_0':
+        'biocontainers/agat:1.2.0--pl5321hdfd78af_0' }"
+
+    input:
+    path gff
```

Comment:
no meta input along with gff ?

Reply:
i figured it wasn't necessary because it deals with references, and people are probably running that only once in their pipeline. i can change it though.

Reply:
added a meta map. let me know how it's looking now. thanks for the detailed review!